src/content/docs/en/reference/publish-to-npm.mdx

```diff
@@ -53,9 +53,9 @@ To create a new package we strongly recommend configuring your development envir
       - ... additional files used by the package
 </FileTree>
 
-In this example, named `my-project`, we create a project with a single package, named `my-component`, and a `demo/` directory for testing and demonstrating the component.
+This example, named `my-project`, creates a project with a single package, named `my-component`, and a `demo/` directory for testing and demonstrating the component.
 
-This is configured in the project root’s `package.json` file.
+The project root’s `package.json` file:
```

Comment:
```suggestion
This is configured in the project root’s `package.json` file:
```
The above lost the helpful sense of configuring this, so I've reverted the text. Approve of the colon!